src/main/java/org/apache/cassandra/sidecar/snapshots/SnapshotPathBuilder.java

```diff
@@ -283,6 +287,13 @@ protected void validate(StreamSSTableComponentRequest request)
         validator.validateTableName(request.tableName());
         validator.validateSnapshotName(request.snapshotName());
         // Only allow .db and TOC.txt components here
+        String indexName = request.indexName();
```

Comment:
secondaryIndexName or siName.

Not to be confused with primary index, which is an sstable component.